### ISSUE TITLE
ECON this job can be used for SCANimate？

### ISSUE THREAD

**Issue by Bill-WangJiLong** [YuliangXiu/ECON#51]
Hello, ECON this job can be used for SCANimate? It looks like their output is the same.

**Comment by YuliangXiu** [comment]
Yes, you can use either the whole ECON (`xxx_full.obj`) or the partial ECON (depth map optimized from d-BiNI) as inputs of SCANimate.

**Comment by Bill-WangJiLong** [comment]
> Yes, you can use either the whole ECON (`xxx_full.obj`) or the partial ECON (depth map optimized from d-BiNI) as inputs of SCANimate.

Thank you for your reply! Can the pose parameters and transl parameters of the smpl model required by SCANimate be obtained directly with this code in infer.py?
if not osp.exists(smpl_obj_path):
                 smpl_obj. export(smpl_obj_path)
                 smpl_info = {
                     "betas":
                     optimal_betas[idx].detach().cpu().unsqueeze(0),
                     "body_pose":
                     rotation_matrix_to_angle_axis(optimed_pose_mat[idx].detach()
                                                  ).cpu().unsqueeze(0),
                     "global_orient":
                     rotation_matrix_to_angle_axis(optimed_orient_mat[idx].detach()
                                                  ).cpu().unsqueeze(0),
                     "transl":
                     optimed_trans[idx].detach().cpu(),
In addition, I also want to ask whether it is necessary to remove some faces in full_obj like ICON does? Or do some changes need to be made to SCANimate?

**Comment by Bill-WangJiLong** [comment]
> 

What should be modified for SCANimate? I noticed that the pose parameter is divided into global rotation and body_pose. Do you mean that it needs to be modified here?

**Comment by ireneisme** [comment]
> > 
> 
> What should be modified for SCANimate? I noticed that the pose parameter is divided into global rotation and body_pose. Do you mean that it needs to be modified here?

Hello, sir! Have you ever finished this piece of work? I am also confused about how to convert the output of ECON which has fewer body_pose to the corresponding smpl model one.
